test/integration/bad/stanc.expected

```diff
@@ -1592,6 +1592,9 @@ Identifier expected after sized type in local (or model block) variable declarat
 
 
   $ ../../../../install/default/bin/stanc lp-error.stan
+Info: integer division implicitly rounds to integer. Found int division: 
+y ^ 2 / 2
+      Positive values rounded down, negative values rounded up or down in platform-dependent way. At 'lp-error.stan', line 5, column 16 to column 19
```

Comment:
Could we maybe move the location to the top line, e.g. "Found int division at %a:"